api/proto/auth/auth.proto

```diff
@@ -11,11 +11,17 @@ message AuthBody {
     bytes  payload = 1;
 }
 
+enum hashAlgorithm {
+    HASH_NONE = 0;
+    HASH_SHA256 = 1;
```

Comment:
If we ever want a different number than 16 bytes than this style of enum names isn't helpful.
I think it makes sense to encode the number of bytes of hash in the hashAlgorithm (and change the comment for senderCertHash to refer to the algo and not 16).

Reply:
makes sense. BTW, roman asked to make this two commits, not combined into this mass commit. I'll move to a different PR #596. Will close this one.